src/styles/blocks/main-content.scss

```
@@ -0,0 +1,9 @@
+.main__wrapper {
+  max-width: 1200px;
```

```suggestion
.main {
  &__wrapper {
  max-width: 1200px;
```
Actually, `main__wrapper` is an element of BEM-block `main`, so it should be written like this and moved to `main.scss` file